lib/metasploit/framework/command/console.rb

```diff
@@ -57,7 +57,6 @@ def start
   def driver
     unless @driver
       # require here so minimum loading is done before {start} is called.
```

Comment:
Looks like this is a stray comment now? 👀

Reply:
sorted